digest/src/lib.rs

```diff
@@ -0,0 +1,40 @@
+pub use multihash;
+use multihash::Multihash;
+
+const SEPARATOR: &[u8; 1] = b":";
+
+pub fn digest(bytes: &[u8]) -> Multihash {
+    // Time the tests and take fastest hash?
+    multihash::Sha3_256::digest(bytes)
```

Comment:
@thomaseizinger @bonomat do we have any strong feeling regarding the hashing function to use for the digest?

Reply:
Not really. I think we can roll with sha256 as you propose and move to something else in the future if needed.